internal/datacoord/channel.go

```diff
@@ -191,7 +191,15 @@ func NewStateChannelByWatchInfo(nodeID int64, info *datapb.ChannelWatchInfo) *St
 	return c
 }
 
-func (c *StateChannel) TransitionOnSuccess() {
+func (c *StateChannel) TransitionOnSuccess(opID int64) {
+	if opID != c.Info.GetOpID() {
```

Comment:
Approve for now. But we definitely need refactor here. We need to
1. Return the Channel status on DN (Watching/Wachted/Releasing/Released/...)
2. Group all info for a given channel on DC to determine the next step

Reply:
@XuanYang-cn